<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationEditor</class>
 <widget class="QWidget" name="SegmentationEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Segmentation Mode</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
        <item>
         <widget class="QComboBox" name="cmbsegmentation_mode"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnResetSegmentation">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="slim5.qrc">
            <normaloff>:/images/loading11.svg</normaloff>:/images/loading11.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="wdgSegmentationButtons" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDoubleSpinBox" name="qsbSegmentationMinArea">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="maximum">
              <double>99999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="qsbSegmentationMaxArea">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="maximum">
              <double>99999.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
           <item>
            <widget class="QDoubleSpinBox" name="qsbSegmentationThreshold">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="maximum">
              <double>99999.990000000005239</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Area (μm²)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Threshold</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Rectangularity</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QDoubleSpinBox" name="qsbSegmentationMinBounds">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="qsbSegmentationMaxBounds">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>ImageJ</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Circularity</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QDoubleSpinBox" name="qsbSegmentationMinCirc">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="qsbSegmentationMaxCirc">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
        <zorder></zorder>
        <zorder>label_3</zorder>
        <zorder>label_2</zorder>
        <zorder></zorder>
        <zorder>label_4</zorder>
        <zorder>groupBox_2</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grbSaveButtons">
     <property name="title">
      <string>Save</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="chkKeepOriginals">
        <property name="text">
         <string>Keep Images</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="slim5.qrc"/>
 </resources>
 <connections/>
</ui>
